src/CONST.ts

```diff
@@ -4313,6 +4313,9 @@ const CONST = {
     SESSION_STORAGE_KEYS: {
         INITIAL_URL: 'INITIAL_URL',
     },
+
+    DOT_SEPARATOR: '•',
```

Comment:
We use this in other places too for preview cards, etc. Just wondering if we can check how we're implementing it elsewhere, and try to consolidate so we don't have multiple ways of implementing this?

Reply:
I've checked it. We use it just in line. I thought, that it would be nice to have it as a constant. But if needed, I can adjust my solution to also use it inline.

Example (`MoneyRequestPreviewContent.tsx`):
![Screenshot 2024-04-11 at 11 53 45](https://github.com/Expensify/App/assets/104823336/75025b57-65e9-4283-af95-ba9eb86347e1)

Reply:
I like using it as a constant, and it would be nice if the other places followed this same approach as well :)

Reply:
Hm, I think I can change it. Do you want me to change it in this PR or create a new one?

Reply:
Follow up is totally fine with me, I don't feel too strongly.

Reply:
I think I can do that here, I will change it

Reply:
> We use this in other places too for preview cards, etc. Just wondering if we can check how we're implementing it elsewhere, and try to consolidate so we don't have multiple ways of implementing this?

@shawnborton  Forgive my ignorance here as I do not understand - `other places too for preview cards`. Can you please point out exactly where you referred to in the FE?

However, I do see usage of `DOT_SEPARATOR` for taxes in the money request flow as shown below.  That way, we do not have multiple ways of implementing this.

![Screenshot 2024-04-11 at 3 58 30 PM](https://github.com/Expensify/App/assets/3004659/945e07e2-b421-4aeb-b34f-8d71e064ca71)

Reply:
I've replaced this character to use the const everywhere except for the `maskCard` function, as it is used in different contexts (not as a separator, but as a mask).

Reply:
Ah I mostly meant that we also use it in places like the expenses detail view:
![CleanShot 2024-04-11 at 15 23 27@2x](https://github.com/Expensify/App/assets/2319350/9fa391e2-c619-4d1b-88b9-1207dd2c86a7)

As well as report previews:
![CleanShot 2024-04-11 at 15 23 51@2x](https://github.com/Expensify/App/assets/2319350/00a80e31-d6de-45ff-9fdf-3d03bd6fa834)

Reply:
I've updated the test sections with the report view. I'm not sure if I will be able to find steps for all the cases.